serverless.yml

```diff
@@ -24,3 +29,5 @@ functions:
     timeout: 60
     events:
       - schedule: rate(1 minute)
+  process-subscriptions:
+    handler: dist/src/handlers/process-subscriptions.handler
```

Comment:
You should definitely add a timeout. Not sure how much though :smile: If the API call processing will take up to 40s this should be done within the remaining 20s. So, let's say 15s to have some time for processing on the main lambda.